Plugin development/General.md

```diff
@@ -1,14 +1,16 @@
-Plugin development doesn't really differ from the theme development. You can use our boilerplate with libs to build a plugin in pretty much the same way you'd build a theme.
+Plugin development is carried out using the [WordPress Plugin Boilerplate](http://wppb.io/). You can find a simple generator for your plugin [here](https://wppb.me/).
 
-Themes usually handle presentational logic, while plugins handle business logic.
+Note that plugins are written in an object-oriented way, whereas the theme is usually written in a procedural way. Object-oriented programming allows for a better separation of concerns.
+
+We are also working on our version of the plugin boilerplate, which can be found [here](https://github.com/infinum/wp-boilerplate-plugin).
```

Comment:
This needs to be replaced with:

`Themes usually handle presentational logic, while plugins handle business logic.`